### ISSUE TITLE
Update runtime transformer to modular core-js

### ISSUE THREAD

**Issue by pdufour** [babel/babel#1206]
Hey thanks for the awesome library. Wanted to bring up a possible issue with using the runtime transformer.

With this code:
````
new Promise();
````

Compiles to:
````
"use strict";

var _core = require("babel-runtime/core-js")["default"];

new _core.Promise();
````
Browserified equals https://gist.github.com/pdufour/61a80fc2f168958d7fb0

Wouldn't be it be more efficient to do
````
"use strict";

var _promise = require('core-js/fn/promise');

new _promise();
````
Browserified equals: https://gist.github.com/pdufour/827fccda20fd4eb7b443

I couldn't be missing something here, let me know. Thanks!

**Comment by sebmck** [comment]
The `["default"]` is done to satisfy other module bundlers that use a `default` property for default imports. Also it wouldn't be `core-js/fn/promise` it'd be `core-js/library/fn/promise` since we don't want the globally polluting one. I'm just waiting on some free time and confirmation from @zloirock that the core-js module transition is in a stable place.

**Comment by zloirock** [comment]
@sebmck it's stable, I do not plan to change it :)

**Comment by sebmck** [comment]
@zloirock Good to know :) I'll rename this issue to update the `runtime` transformer to reflect it then.

**Comment by sebmck** [comment]
Ping @stefanpenner

**Comment by sebmck** [comment]
Released as of 5.1.0, thanks!

**Comment by stefanpenner** [comment]
> Ping @stefanpenner

:+1:
